pkgs/development/python-modules/arelle/default.nix

```diff
@@ -1,32 +1,32 @@
 { gui ? true,
   buildPythonPackage, fetchFromGitHub, lib,
   sphinx_1_2, lxml, isodate, numpy, pytest,
-  tkinter ? null, py3to2, isPy3k,
+  tkinter ? null, isPy3k, pythonAtLeast,
   ... }:
 
 buildPythonPackage rec {
   name = "arelle-${version}${lib.optionalString (!gui) "-headless"}";
-  version = "2017-08-24";
+  version = "2017-10-07";
 
-  disabled = !isPy3k;
+  # Import semantics have changed in python 3.5. Upstream uses python 3.4.
+  disabled = !isPy3k && (!pythonAtLeast "python35");
```

Comment:
It stills tries to build for python 3.6 on my system. How about using `isPy34` instead?

Reply:
Thank you. I should have thought a bit more deeply about all those negations... I'm going for `isPy34` instead. I didn't like `pythonAtLeast` anyway.